axon_schema_test.go

```diff
@@ -0,0 +1,26 @@
+package warppipe
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseSemver(t *testing.T) {
+	for _, tc := range []struct {
+		raw   string
+		major int
+		minor int
+	}{
+		{raw: "9.5", major: 9, minor: 5},
+		{raw: "9.5   (And some other stuff...", major: 9, minor: 5},
```

Comment:
i'd add a test for like `11.43` too, to be sure it can handle >1 character version parts

Reply:
fadbd7ad71871bac642e773374d60066839f4519